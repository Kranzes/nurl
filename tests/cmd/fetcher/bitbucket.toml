args = ["https://bitbucket.org/asbuch/lrcalc", "lrcalc-2.0"]
