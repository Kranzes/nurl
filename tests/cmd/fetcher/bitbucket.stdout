fetchFromBitbucket {
  owner = "asbuch";
  repo = "lrcalc";
  rev = "lrcalc-2.0";
  hash = "sha256-9u2tNAp1DDPHu7FtFZWDrVkqCeFCyz0RI8oXmmSwj5I=";
}